.gitpod.yml

```diff
@@ -4,7 +4,19 @@ image:
 checkoutLocation: dazzle
 workspaceLocation: dazzle
 tasks:
+  - command: sudo buildkitd --group gitpod
+  - command: docker run -p 5000:5000 registry
+    openMode: split-right
   - init: go get -v ./...
+    openMode: tab-after
+    command: |
+      gp await-port 5000
+      go generate -v ./...
+      go run main.go build --context example $(gp url 5000 | cut -d/ -f3)/dazzle
+ports:
+  - port: 5000
```

Comment:
Why make it public? 
localhost:5000/dazzle doesn't require opening it up.

Reply:
Docker runs in its own network namespace. I doubt `localhost:5000` would work

Reply:
WFM for the last 2/3 days 🤷

Reply:
It does work - I stand corrected :) 

Now I'm left wondering why it works 🤷